datamodels/2.x/itop-backup/dictionaries/zh_cn.dict.itop-backup.php

```diff
@@ -3,45 +3,59 @@
  * Localized data
  *
  * @copyright Copyright (C) 2010-2024 Combodo SAS
- * @license    https://opensource.org/licenses/AGPL-3.0
- * 
- */
-/**
+ * @license    http://opensource.org/licenses/AGPL-3.0
```

Comment:
Please revert to https:
```suggestion
 * @license    https://opensource.org/licenses/AGPL-3.0
```
(Also on all other files)

Reply:
plz update en.dict first

Reply:
Why is that relevant? It was correct on this file before and you changed it.